components/expand-collapse/README.md

```diff
@@ -0,0 +1,37 @@
+# Expand Collapse
+
+The `d2l-expand-collapse` element can be used to used to create expandable and collapsible sections. This component only provides the logic to expand and collapse the section. Controlling when and how the section expands or collapses is the responsibility of the user.
+
+![More-Less](./screenshots/expand-collapse.gif?raw=true)
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/expand-collapse/expand-collapse.js';
+</script>
+
+<button>Toggle</button>
+<d2l-expand-collapse>
+  <p>My expand collapse content.</p>
+</d2l-expand-collapse>
+
+<script type="module">
```

Comment:
I don't think this script (or the button HTML) is really needed in the sample code.  The property is already listed below, so this doesn't _really_ add anything.  I would consider adding `expanded` attribute in the above HTML snippet though.

Reply:
The only reason I included it was to try and give users an example of a correct usage with `aria-expanded` but I'm happy to trim it down since thats explained in the README.

Reply:
Sorry, I skimmed over the aria part.  🤷‍♂️ Either way I guess.